Lombiq.VueJs.Samples/Lombiq.VueJs.Samples.csproj

```diff
@@ -1,11 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk.Razor">
 
   <Import Condition="'$(NuGetBuild)' != 'true'" Project="..\..\..\Utilities\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions.props" />
-  <Import Project="..\..\Lombiq.VueJs\Lombiq.VueJs\Lombiq.VueJs.props" />
+  <Import Project="..\Lombiq.VueJs\Lombiq.VueJs.props" />
 
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
     <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
+    <DefaultItemExcludes>$(DefaultItemExcludes);.git*;node_modules\**</DefaultItemExcludes>
```

Comment:
In one of the many MSBuild issue threads I've read over the past year, an MS dev mentioned that additional `DefaultItemExcludes` should rather be added to the beginning of this expression.

Also, the `node_modules\**` entry is superfluous as it is already included - see the [NE.csproj](https://github.com/Lombiq/NodeJs-Extensions/blob/dev/Lombiq.NodeJs.Extensions/Lombiq.NodeJs.Extensions.csproj#L5).

Reply:
That's how it is in all of our projects, according to the instructions [in the wiki](https://lombiq.atlassian.net/wiki/spaces/DEV/pages/19496967/Creating+a+new+Visual+Studio+project+or+solution).
![image](https://user-images.githubusercontent.com/4007293/217820971-4f569ae6-1208-4a98-b9d5-f4acea247bf5.png)
If you think this is incorrect then imo it should be updated everywhere at once, in a separate issue.

> Also, the node_modules\** entry is superfluous as it is already included - see the [NE.csproj](https://github.com/Lombiq/NodeJs-Extensions/blob/dev/Lombiq.NodeJs.Extensions/Lombiq.NodeJs.Extensions.csproj#L5).

I think this is a project level setting, so it shouldn't inherit from NE's project file.